tests/gis_tests/geoapp/tests.py

```diff
@@ -711,10 +711,16 @@ def test_unionagg_tolerance(self):
         tx = Country.objects.get(name="Texas").mpoly
         # Tolerance is greater than distance between Forney and Dallas, that's
         # why Dallas is ignored.
-        forney_houston = GEOSGeometry(
-            "MULTIPOINT(-95.363151 29.763374, -96.467222 32.751389)",
-            srid=4326,
-        )
+        if connection.ops.oracle and connection.oracle_version >= (23,):
+            forney_houston = GEOSGeometry(
+                "MULTIPOINT (-95.363151 29.763374, -96.801611 32.782057)",
```

Comment:
Marking this PR as draft. 

This is the edit to make the test pass, but it seems wrong to me. This new point is Dallas (entering the point at https://www.geometrymapper.com/), before it was Forney. Note above would suggest this change is wrong. 

Thoughts welcome 😄

Reply:
Maybe it's related with undefined ordering :thinking:

Reply:
Taking another look at this morning. Just leading a small note here for now to say that I've just realised the `supports_tolerance_parameter` feature (and therefore this test) is only enabled on Oracle.